github/libp2p.yml

```diff
@@ -4568,15 +4648,11 @@ repositories:
     squash_merge_commit_title: PR_TITLE
     teams:
       admin:
-        - Admin
-        - rust-libp2p Active Maintainers
-        - w3dt-stewards
-      maintain:
-        - rust-libp2p Maintainers
+        - sigma-prime
```

Comment:
@AgeManning Thanks for creating this PR but my 2 cents is that I like the abstraction that rust-libp2p Maintainers preserves. If a new full time rust-libp2p maintainer is at Shipyard or other org that's not Sigma Prime, it wouldn't be correct to add that new maintainer to the sigma-prime user group; and because they are a full time maintainer, it wouldn't be correct to add them to rust-libp2p-contributors either.

Wdyt about keeping rust-libp2p Maintainers and having that group consist solely of sigma-prime users?

Reply:
Yeah for sure. 

We are adding new people from sigp in the near future. I thought it might be useful to group them under a `sigma-prime` banner so it was easier to identify people and change if we no longer become maintainers. 

More than happy just to keep it general and rename to `rust-libp2p-maintainers`.